Makefile.PL

```diff
@@ -21,6 +21,7 @@ build_requires( 'Test::More' => 0, );
 requires(
     'JSON::XS'           => 0,
     'Locale::TextDomain' => 1.23,
+    'Net::Interface'     => 0,
```

Comment:
I can't find a binary package for this on CentOS. Have you checked for alternatives?

Reply:
@mattias-p  I only found Net::Interface that can handle with IPv4 and IPv6. IO::Interface for example does not work with IPv6. Any advices for an alternate library ?

Reply:
At least for CentOS 7 I find a package: https://centos.pkgs.org/7/openfusion-x86_64/perl-Net-Interface-1.012-2.of.el7.x86_64.rpm.html

Reply:
According to https://rt.cpan.org/Public/Bug/Display.html?id=124582, me should add this somewhere in travis.yml
`mkdir /usr/include/sys/; ln -s /usr/include/bits/socket.h /usr/include/sys/socket.h` to prevent the fail on Travis.

Reply:
@vlevigneron Thanks for the clarification regarding the choice of library. And no, I don't have any alternatives up my sleeve.